joblib/test/common.py

```diff
@@ -32,6 +33,12 @@ def my_func():
         return my_func
     np = None
 
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
+with_pandas = pytest.mark.skipif(not pd, reason='Test requires pandas.')
```

Comment:
You might want to replace those lines with this one and place it in the relevant tests.
```suggestion
pd = pytest.importskip("pandas")
```

Reply:
Let's stay consistent with `with_numpy` fixture and keep it as is.